eclipsePlugin/doc/building_spotbugs_plugin.txt

```diff
@@ -41,9 +41,10 @@ Import the projects
 8) Choose [File | Import General | Existing Projects into Workspace | Next]. The Import project import wizard opens.
 9) In the wizard, proceed as follows:
    a) Choose [Select root directory].
-   b) Enter ~/git/spotbugs. Click [Browse]
-   c) Click [Finish].
-   g) Now "spotbugs" project will be compiled by Eclipse and should not have any errors,
+   b) Enter ~/git/spotbugs. Click [Browse]. Make sure the checkbox "Search for nested projects" is selected.
```

Comment:
@henrik242 Shouldn’t we clone to a different directory (say `spotbugs-project`) instead? AFAICT, the above does not make **all** projects available in Eclipse, which is not a good basis to start contributing on SpotBugs; you miss out on important files like the root `build.gradle`.

(FWIW, I’ve opened a bug with the Eclipse Buildship project about this problematic import behavior a while ago: [Bug 518089](https://bugs.eclipse.org/bugs/show_bug.cgi?id=518089))